nuocd/nuodb_adminquery.py

```diff
@@ -114,12 +116,26 @@ def get_admin_conn(pid):
                   default=10,
                   type=int,
                   help='interval in seconds between measurements.')
+parser.add_option('-q',
+                  '--query-admin',
+                  dest='query_admin',
+                  default=False,
+                  action='store_true',
+                  help='interval in seconds between measurements.')
```

Comment:
It might be descriptive enough,  that said,  I will be removing it, as this is change relative to fix that is not part of python3 port.  This was accidentally included in this branch.